src/MoBi.Core/Snapshots/Mappers/ProjectMapper.cs

```diff
@@ -105,12 +112,15 @@ private async Task<ModelProject> mapToModel(SnapshotProject projectSnapshot, Pro
          NumberOfSimulationsLoaded = 0
       };
 
+      var sims = new List<MoBiSimulation>();
```

Comment:
I guess we could get away without this variable. We only use it to tally the simulations, but we are going to run all the simulations in the project after this step so we could just get it from there.